app/models/seller.rb

```diff
@@ -35,6 +35,7 @@ class Seller < ApplicationRecord
   # The `seller_id` of the special seller we use to collect pool donations.
   POOL_DONATION_SELLER_ID = 'send-chinatown-love'
 
+  # NOTE: POOL_DONATION_SELLER_ID should not accept donations
```

Comment:
This already exists - is this just a note to acknowledge that? maybe s/should/does?

Reply:
yeah this is because as i was reading through the code it wasn't apparent to me, so wanted to make it clear here

Reply:
That makes sense to me - can we rephrase as "does not accept donations"